src/nxdoc/web-ui/web-ui-tutorials/web-ui-bulk-actions.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Bulk Actions in Web UI'
+review:
+    comment: ''
+    date: '2021-09-07'
+    status: ok
+toc: true
+labels:
+    - nuxeo-web-ui
+    - bulk
+tree_item_index: 490
+---
+
+## Enabling Bulk Actions
+
+{{#> callout type='info'}}
+Bulk actions leverage the [Bulk Action Framework]({{page page='bulk-action-framework'}}). Make sure your [architecture]({{page page='nuxeo-reference-architectures'}}) complies with it.
+{{/callout}}
+
+Bulk actions are available since LTS 2021. To prevent bringing any breaking change through an update, you need to enable this feature explicitly by adding the following property to your [nuxeo.conf]({{page page='configuration-parameters-index-nuxeoconf'}}) file:
+```
+nuxeo.selection.selectAllEnabled=true
+```
+
+## Using Bulk Actions
+
+Once enabled, users can select all documents in a results listing and execute an action on them. A selection can be done while browsing content and in the context of search results. It is possible to select all documents after using filters as well to fine tune the results first. Note that adding filters, changing the display or the sort order for the results causes the current selection to be lost: this results in sending a different query to the server and we cannot keep track of the difference between the two without hurting performance.
+
+//TODO image select all and default bulk actions
+
+//TODO image select all after filtering
+
+Clicking on the checkbox selects all documents at once, both the ones that are visible and the ones that are not currently displayed on screen. Clicking it again deselects all documents. Note that it is not possible to deselect documents individually at this stage.
```

Comment:
What checkbox? The one in the header of the listing? If so, we should make it clear.
```suggestion
Clicking on the checkbox of the header selects all documents at once, both the ones that are visible and the ones that are not currently displayed on screen. Clicking it again deselects all documents. Note that it is not possible to deselect documents individually at this stage.
```